girder/api/v1/user.py

```diff
@@ -260,28 +260,35 @@ def changeUserPassword(self, user, params):
     def changePassword(self, params):
         self.requireParams(('old', 'new'), params)
         user = self.getCurrentUser()
+        token = None
 
         if not params['old']:
             raise RestException('Old password must not be empty.')
 
         if (not self.model('password').hasPassword(user) or
```

Comment:
A user with no password is currently not able to use this endpoint -- the OAuth plugin and any other plugin that allows passwordless users to exist should really hook into this and do something sensible, but as of right now that doesn't exist.